web/src/routes/(welcome_pages)/+layout.svelte

```diff
@@ -2,20 +2,29 @@
     import Header from '$lib/components/Header.svelte';
     import Footer from '$lib/components/Footer.svelte';
     import '../../app.css';
-    import Toast from "$lib/components/Toast.svelte";
+    import { Toaster } from '$lib/components/ui/sonner';
     import {current_user} from "$lib/state/current_user.svelte";
+    import { afterNavigate } from '$app/navigation';
+    import { checkAndShowPendingToasts } from '$lib/state/toast.svelte';
 
     let { children, data } = $props();
 
     current_user.id = data.current_user_id;
 
+    afterNavigate(() => {
```

Comment:
[Just FYI, not feedback]
I have routes/(app) and routes/(welcome_pages) to distinguish between the signed in & signed out experience, but I think we're near the future where the lines will be blurred too much & we can merge the two experiences and check if the user is signed in.
For example, I want to be able to share a link to an event with my friends who are not currently on the app.

Reply:
Yeah makes sense 🤙🏼. I think we talked about having `AuthProvider` or `AuthContext` system which would be nicer.